Emacs/README.md

```diff
@@ -109,7 +110,7 @@ Spacemacs can be used naturally by both Emacs and Vim users -- you can even mix
 
 1. If you have an existing Emacs configuration, back it up first:
 
-   ```sh
+   ```console
```

Comment:
I would like to keep this as `console` since this is a pre install command. Let me know if you have a differing opinion.. will change it. For now i would like to maintain this idea.